lib/index.js

```diff
@@ -126,6 +155,7 @@ Ramen.prototype.identify = function(identify){
 
   // For tracking purposes
   ramenSettings._partner = 'segment.com';
+  ramenSettings.disable_location_watch = true;
```

Comment:
Seems like we'd want this in `#initialize` rather than in `#identify`. `#page` gets called immediately on pageload.

But now I'm realizing that this raises the bigger issue of `window.ramenSettings` not existing until `#identify` has been called, which is no longer guaranteed given the above point.

I'd say we should move as much of the `ramenSettings` logic into `#initialize` as possible, and then inside of `#identify` we can augment it with user data (like you're doing above).

Or does Ramen consider a `#page` call useless until `#identify` has been called? Do you want to prevent `#page` calls from going through until `#identify` has been called? I'm not familiar enough with how Ramen works so maybe you can detail the conditions under which each call should occur.